nomad/rpc.go

```diff
@@ -84,6 +84,7 @@ type RPCContext struct {
 // listen is used to listen for incoming RPC connections
 func (r *rpcHandler) listen(ctx context.Context) {
 	defer close(r.listenerCh)
+	var tempDelay time.Duration
```

Comment:
I think this needs to be a more descriptive name. `acceptLoopDelay` or something along those lines